bukuserver/static/bukuserver/js/page_size.js

```diff
@@ -0,0 +1,20 @@
+$(document).ready(function() {
+  const SIZES = [20, 50, 100]; // hardcoded list; see page_size_form in admin/model/layout.html
```

Comment:
add 10 to this list (default page size)

Reply:
The reason I'm hardcoding _these particular values_ is specifically because these same values are [hardcoded in flask-admin](https://github.com/flask-admin/flask-admin/blob/v1.6.0/flask_admin/templates/bootstrap3/admin/model/layout.html#L103-L105). (…And yes, this actually appears to be the best way to identify the page-sizes dropdown at runtime.)

Reply:
from short test i find that i have to go to other page and back to bookmark page again to set oage_size to default 10

i also tried to add 10 this const var but it is not working

if it is harder to add default 10 page_size, i will let it pass and keep the const var as it is

Reply:
To set page size to default (i.e. _unset_ it), [just enter empty value in the custom size prompt dialogue](https://github.com/LeXofLeviafan/buku/blob/e1bfa897fe7e768261170a62d83e6d62f8c5bf8e/bukuserver/static/bukuserver/js/page_size.js#L9).
![modified menu](https://user-images.githubusercontent.com/16271003/205191948-2266fbf4-2893-4afc-9b38-b7202e67e465.png "click \"custom\"")
![prompt dialogue](https://user-images.githubusercontent.com/16271003/205192203-fb7f40df-c2f8-4677-b690-d6743f94f943.png "then hit Backspace and Enter")

And no, the `SIZES` value _shouldn't_ be modified (see my previous comment, and particularly [the link](https://github.com/flask-admin/flask-admin/blob/v1.6.0/flask_admin/templates/bootstrap3/admin/model/layout.html#L103-L105)).

Reply:
Ok then I think this is resolved